nautobot/utilities/utils.py

```diff
@@ -689,3 +720,220 @@ def pretty_str(self, node=None, depth=0):
 
     # Use pretty_str() as the string generator vs. just stringify the `Q` object.
     return pretty_str(query)
+
+
+def build_lookup_label(field_name, verbose_name):
+    """
+    Return lookup expr with its verbose name
+
+    Args:
+        field_name (str): Field name e.g slug__iew
+        verbose_name (str): The verbose name for the lookup exper which is suffixed to the field name e.g iew -> iendswith
+
+    Examples:
+        >>> build_lookup_label("slug__iew", "iendswith")
+        >>> "ends-with(iew)"
+    """
+
+    label = ""
+    search = re.search(r"(?<=__)\w+", field_name)
```

Comment:
Please move this `re` pattern to a module-level constant and use `re.compile` so that it is runtime-optimized, and then you can use the pattern to call `.search` here in the business logic.

e.g.

```python
# Defined at top of module..
SEARCH_RE = re.compile(r"(?<=__)\w+")

# later in code...
def build_lookup_table(field_name, verbose_name): 
    # Call it like this:
    search - SEARCH_RE.search(field_name)
```